Makefile

```diff
@@ -926,31 +924,6 @@ init-submodules-e: init-webapps-submodules-e
 	git submodule init e
 	git submodule update
 
-# Update go.mod and vendor files.
-.PHONY: update-vendor
-update-vendor:
-	# update modules in api/
-	cd api && go mod tidy
-	# update modules in root directory
-	go mod tidy
```

Comment:
i think running [`go mod tidy`](https://go.dev/ref/mod#go-mod-tidy) may still be beneficial to update/fix go.mod and go.sum

Reply:
Agree - but we don't need a make target for that. Just like we don't have a make target for `go get` when we want to add or update a dependency. Just use the standard `go` commands directly for operations on go.mod.

Reply:
Yes. i think i was thinking more about the cases where go.mod/sum messed up (like maybe someone manually did something to these files or bad merges). Good thing now go build will fail in those cases. So maybe it's better this way so that CI build fails and that forces ppl to correct them.